packages/next-templates/src/app/styling/css/productpagina-klachtenformulier.css

```diff
@@ -0,0 +1,8 @@
+/* BREADCRUMNAV */
+.breadcrumb-nav {
+  padding: var(--voorbeeld-space-block-rabbit);
```

Comment:
Om afstand te maken tussen componenten is `margin` meestal de juist keuze in plaats van padding.
Klopt het dat je op alle assen ruimte wilt toevoegen? Of alleen `margin-block-start` en `margin-block-end`?

Reply:
Correct. Dat had ik moeten weten ook, is ons volgens mij in jaar 1 al verteld op FDND. 🤦🏼‍♀️ 
Aangezien ik hier padding gebruik, is de kans aanwezig dat ik dit op andere pagina's ook gedaan heb. Voor ik daar PR's voor aanmaak zal ik daar eerst even naar kijken!

Ik heb de padding vervangen voor margin. 
Heb gelijk ook de classname aangepast naar voorbeeld-breadcrumb-nav.